ils_middleware/tasks/folio/mappings/bf_instance.py

```diff
@@ -1,16 +1,3 @@
-date_of_publication = """PREFIX bf: <http://id.loc.gov/ontologies/bibframe/>
-PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-
-SELECT ?date
-WHERE {{
-    <{bf_instance}> a bf:Instance .
-    <{bf_instance}> bf:provisionActivity ?activity .
-    ?activity a bf:Publication .
-    ?activity bf:date ?date .
-}}
-"""
-
 identifier = """PREFIX bf: <http://id.loc.gov/ontologies/bibframe/>
```

Comment:
I would appreciate some documentation in this file about it's purpose.

Reply:
Added module descriptions to both `folio.mappings.bf_instance` and `folio.mappings.bf_work`.